arduino/WandCore/TouchSurfaceClient.cpp

```diff
@@ -0,0 +1,115 @@
+/*
+  Capacitive Touch Surface Client
+
+  Manages a Touch Surface Input
+*/
+
+#include "TouchSurfaceClient.h"
+
+// Pin the touch surface is attached to
+int touchPin;
+
+// Touch surface constants
+const int touchThold = 50;                  // readings above mean the sensor is not being touched
+const int debounce = 20;                    // ms debounce period to prevent flickering when touching or stopping to touch the surface

```

Comment:
> or stopping to touch the surface

Maybe word this as "releasing from the surface"?

Reply:
If you want to be consistent with later uses, maybe "stopping of touching the surface".